k8s/cmd/admission-webhook/const.go

```diff
@@ -12,6 +12,8 @@ const (
 	repoEnv                      = "REPO"
 	initContainerEnv             = "INITCONTAINER"
 	tagEnv                       = "TAG"
+	jaegerHostEnv                = "JAEGER_SERVICE_HOST"
+	jaegerPortEnv                = "JAEGER_SERVICE_PORT_JAEGER"
```

Comment:
Is there a reason this is "JAEGER_SERVICE_PORT_JAEGER" instead of "JAEGER_SERVICE_PORT"?

Reply:
That's the value being used in the InitJaeger() function in pkg/tools/tools.go.  So, it needs to match that.

https://github.com/networkservicemesh/networkservicemesh/blob/master/pkg/tools/tools.go#L180

Reply:
Do we want to name JAEGER here, or are we really setting the HOST/PORT for sending OpenTracing information?

Reply:
@tiswanso Got it.  Makes sense... principle of least change.  If we want to change that... that's a different effort than this :)

Reply:
Good question... definitely the current tools impl is opinionated on what is the tracing collector--ie. 
- https://github.com/networkservicemesh/networkservicemesh/blob/master/k8s/cmd/nsmd-k8s/main.go#L29
- https://github.com/networkservicemesh/networkservicemesh/blob/master/side-cars/cmd/nsm-init/main.go#L32
- https://github.com/networkservicemesh/networkservicemesh/blob/master/sdk/endpoint/endpoint.go#L73

That seems to be a common pattern in general (not necessarily specific to NSM) for golang.  Maybe there's a more opentracing generic way to code it complete with config settings but I think that'd be something to investigate and follow-on if desired.